python/tvm/autotvm/measure/local_executor.py

```diff
@@ -145,6 +145,7 @@ def submit(self, func, *args, **kwargs):
         if not self.do_fork:
             return LocalFutureNoFork(func(*args, **kwargs))
 
+        # TODO why they choose a queue size of 2? add a comment
```

Comment:
A quick git blame points to this: https://github.com/apache/incubator-tvm/pull/1651

@merrymercy made the change with "The old queue size is too small. It will stall the executor due to race condition." as comment

Reply:
I added a comment to this effect